l10n_nl_tax_declaration_reporting/__openerp__.py

```diff
@@ -0,0 +1,62 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Dutch accounting taxes addon
+#    Copyright (C) 2014 ONESTEiN BV (<http://www.onestein.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'VAT Report Netherlands',
+    'version': '1.0',
+    'category': 'Localization',
+    'sequence': 1,
+    'description': """
+This module provides you with the Tax Statement in the Dutch format.
+In the report you will find the taxes charged to customers and taxes that
+suppliers charge to you. The layout this module provides will seem very
+familiar to everyone wanting to declare taxes in The Netherlands.
+
+The standard Tax Declaration report is not replaced by this module, a new
+report with the Dutch layout is added to the menu
+Reporting > Generic Reporting > Taxes.
+
+--
+
+Deze module biedt u de BTW aangifte in Nederlands formaat.
+In het rapport staat de BTW berekend aan klanten, en de btw die
+leveranciers aan u berekenen. Het rapport is met deze indeling zeer
+herkenbaar voor iedereen die een belasting aangifte wilt doen.
+
+Dit rapport verwijdert niet de standaard, maar wordt toegevoegd in
+het menu Rapportages > Algemene rapporten > Belastingen.
+    """,
+    'author': 'ONESTEiN BV',
+    'website': 'http://www.onestein.nl',
+    'depends': ['account',
+                'base_vat',
+                'base_iban',
+                'account_chart',
+                'l10n_nl',
```

Comment:
Agree but the report depends on the indicator (BTW) or (omzet) on the base_code_id. Manual managed tax codes should take this into account. Adding an indicator for either a revenue tac-code or an tax tax-code. But this might have more impact or at least an agreement on the roadmap for using this feature like adding a filter on the base.code.id and tax.code.id fields in account.tax.form